app/assets/javascripts/discourse/app/controllers/create-account.js

```diff
@@ -52,27 +54,6 @@ export default Controller.extend(
       return (hasAuthOptions || canCreateLocal) && !skipConfirmation;
     },
 
-    resetForm() {
-      // We wrap the fields in a structure so we can assign a value
-      this.setProperties({
```

Comment:
How come we're making this change in the create-account modal? This one is still a legacy controller, so I would've thought that we still need the manual reset?